### PR TITLE
Make sure actor started timer before we change time

### LINKED ISSUE #25799: Failed: ManualTimerExampleSpec
https://jenkins.akka.io:8498/job/akka-nightly/5961/consoleFull

### PR DESCRIPTION
**Pull request by johanandren**
Fixes  #25799

### CODE REVIEW

**Review by raboof — APPROVED** [review]

File: akka-actor-testkit-typed/src/test/scala/docs/akka/actor/testkit/typed/scaladsl/ManualTimerExampleSpec.scala [hunk 1]
```diff
@@ -77,21 +78,25 @@ class ManualTimerExampleSpec extends ScalaTestWithActorTestKit(ManualTime.config
...
+            timer.startPeriodicTimer("T", Tick(nextCount), interval)
+            probe.ref ! SlowThenBumpAck
+            Behaviors.same
         }
       }
 
       val ref = spawn(behavior)
+
+      // make sure we actually started the timer before we change the time
+      probe.expectMessage(Started)
```
Comment (helena) [inline akka-actor-testkit-typed/src/test/scala/docs/akka/actor/testkit/typed/scaladsl/ManualTimerExampleSpec.scala]: This is the crux right? The rest is mostly formatting and minors.

**Review by helena — APPROVED** [review]
LGTM

File: akka-actor-testkit-typed/src/test/scala/docs/akka/actor/testkit/typed/scaladsl/ManualTimerExampleSpec.scala (hunk 1 above)
Comment (johanandren) [reply akka-actor-testkit-typed/src/test/scala/docs/akka/actor/testkit/typed/scaladsl/ManualTimerExampleSpec.scala]: Yep! Compiler warnings mostly, the rest.

**Comment by johanandren** [comment]
Travis is in fact green, not sure why it is marked as still pending.
https://travis-ci.org/akka/akka/builds/507992669?utm_source=github_status&utm_medium=notification

**Pull request merged**